ansible/roles/apps/stubbornjava/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+  supervisor:
+    app_name: stubbornjava
+    app_directory: /home/ec2-user/apps/stubbornjava
+    app_command: java8 -Denv=prod -Xmx640m -cp 'stubbornjava-all.jar' com.stubbornjava.webapp.StubbornJavaWebApp
```

Comment:
The java8 role doesn't default to java8 for `java` but I kind of like this explicit way more so I didn't bother fixing the default.